stix_shifter_modules/aws_athena/stix_translation/json/ocsf_from_stix_map.json

```diff
@@ -637,19 +637,19 @@
         "identity.authorizations.policy.desc"
       ],
       "authorizations.policy_group_desc": [
-        "identity.authorizations.policy.group.desc"
+        "'identity.authorizations.policy.group.desc'"
```

Comment:
why did you add extra single quotes?

Reply:
"group" is considered a special keyword in SQL. If you try to query this field without quotes, it will always fail because it's going to interpret it as the keyword. Putting quotes around it, to my knowledge, allows sql to interpret it as intended (as a an object in the table). 

I admittedly may need more testing on this one, I just checked that the query doesn't error out after the change.